src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Form\Bridge\Symfony\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * This is the class that validates and merges configuration from your app/config files.
+ *
+ * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ * @author Alexander <iam.asm89@gmail.com>
+ */
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('sonata_form');
+
+        $this->addFlashMessageSection($rootNode);
+
+        $rootNode
+            ->children()
+                ->arrayNode('form')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('mapping')
```

Comment:
Form mapping was deprecated in `3.x`. May be this block should be removed from here?

Reply:
But this is still used in `3.x` isn't it?

Reply:
Removed the mapping, is the node isn't used anymore.